File: cakk-api/src/main/java/com/cakk/api/dto/response/shop/CakeShopByMapResponse.java
package com.cakk.api.dto.response.shop;

import java.util.List;

import com.cakk.domain.mysql.dto.param.shop.CakeShopByLocationParam;


public record CakeShopByMapResponse(
	List<CakeShopByLocationParam> cakeShops
) {
}

File: cakk-core/src/main/kotlin/com/cakk/core/dto/response/shop/CakeShopByMapResponse.kt
package com.cakk.core.dto.response.shop

import com.cakk.infrastructure.persistence.param.shop.CakeShopLocationResponseParam

data class CakeShopByMapResponse(
    val cakeShops: List<com.cakk.infrastructure.persistence.param.shop.CakeShopLocationResponseParam>
)

